README.md

```diff
@@ -18,16 +18,17 @@ To setup on your local machine:
     ```
```

Comment:
In the above, "more details on setting up your machine locally, on Spark, or on Azure Databricks.", I was expecting to go to Azure Databricks page when I click the link, not our setup steps page. But anyway it is fine... just wonder why only Azure Databricks has a link, not Spark?